docs/devices-api/sdks/connect-widget/react-native.mdx

```diff
@@ -5,10 +5,17 @@ description: "A library to integrate with Metriport on React Native - including
 
 <Info>
   If you haven't already, and you're developing on iOS, visit the [Apple Health
-  docs](/devices-api/getting-started/apple-health#1-enable-healthkit) and follow the "Enable HealthKit"
-  section to configure HealthKit.
+  docs](/devices-api/getting-started/apple-health#1-enable-healthkit) and follow the "Enable
+  HealthKit" section to configure HealthKit.
 </Info>
 
+<Warning>
+  If you're using Expo Go, you'll need to prebuild your app to use this library. You can read more
+  about this [here](https://docs.expo.dev/workflow/customizing/). **Note that Apple Health lives on
+  the user's device, and requires the native bridge to work. So, Apple Health integrations simply
```

Comment:
Seems like [it depends](https://english.stackexchange.com/questions/30436/when-do-we-need-to-put-a-comma-after-so-at-the-beginning-of-a-sentence) - I put the comma there since it flows better with a pause, since it's a prominent statement that things "simply won't work"